City-of-Helsinki/youth-membership-admin-ui
# YM-416 | Fix profile editing

## Description
These changes adjust profile editing mutations so that they work with the new federated API.

I also added s more strict check for verifying that tokens are correct.

I had to refactor settings for browser tests a bit because the same data is not available as is in the Helsinki profile test API.

I made the HasPermission check checks both services (Helsinki profile and Jassari) because it turns out that the user needs special groups configured in both services.

## Context
These changes make it possible for the new federated API to be used with this UI.

[YM-416](https://helsinkisolutionoffice.atlassian.net/browse/YM-416)

## How Has This Been Tested?
I refactored the testing script to work with configurable settings and ran the fixtures for editing and viewing. Both passed.

## Manual Testing Instructions for Reviewers
As an admin level user

1. Go to profiles
1. Find the profile with name "Kissa Bistoksissa" (others likely won't work)
1. Check that you can view the information as expected
1. Check that you are able to edit information as expected
